### DISCUSSION TITLE
Change App Title

### DISCUSSION THREAD

**Discussion by Theinventor17** [danny-avila/LibreChat#2300] [Help Wanted]
I'm trying to change the site title for the login page. It seems once i compile it on Docker it still has the generic "Librechat" title but i changed the .ENV File to the new title and it only changes to the new title if i log in to the program. How do i change the title of the login  and registration page?

**Comment by fuegovic** [comment]
you would need to manually edit the code in the index.html files for that

**Reply by Theinventor17** [reply]
I did change the index.html file is there a specific index file i should be looking for? Like in a specific folder

**Reply by fuegovic** [reply]
there's a second one in /client

if you're using the docker image the changes won't apply, you need to use either npm or build the docker image yourself to see the changes
